app/Configs.php

```diff
@@ -8,95 +8,80 @@
 
 class Configs extends Model
 {
+	/**
+	 *  this is a parameter for Laravel to indicate that there is no created_at, updated_at columns.
+	 */
 	public $timestamps = false;
-	private static $public_cache = null;
 
-	protected static $except = [
-		'username',
-		'password',
-		'dropboxKey'
-	];
 
-	/**
-	 * @param string $key
-	 * @param bool $public
-	 * @return bool
-	 */
-	static protected function inExceptArray(string $key, bool $public)
+	private static $cache = null;
+
+
+
+	public static function arrayify($query)
```

Comment:
Eloquent does something for this: https://laravel.com/docs/5.8/collections#method-pluck

Reply:
Niiiiice :astonished:  Thanks !